server/repository/src/migrations/v2_14_00/resync_existing_vaccination_encounter_records.rs

```diff
@@ -0,0 +1,40 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "resync_existing_vaccination_encounter_records"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        sql!(
+            connection,
+            r#"
+                INSERT INTO changelog (table_name, record_id, row_action, name_link_id)
+                SELECT
+                    'vaccination',
+                    id,
+                    'UPSERT',
+                    patient_link_id
+                FROM vaccination;
+            "#
+        )?;
+
+        sql!(
+            connection,
+            r#"
+                INSERT INTO changelog (table_name, record_id, row_action, name_link_id, store_id)
+                SELECT 
+                    'encounter',
+                    id,
+                    'UPSERT',
+                    patient_link_id,
+                    store_id
+                FROM encounter;
+            "#
+        )?;
```

Comment:
Thought (I presume thought is even lower request than nit? haha): Doesn't matter but you could do one `sql!` expression with both sql statements in it. Though maybe it's better this way as it'll be completely unambiguous which statement failed!